client/my-sites/checkout/controller.jsx

```diff
@@ -48,8 +47,7 @@ const debug = debugFactory( 'calypso:checkout-controller' );
 export function checkout( context, next ) {
 	const { feature, plan, purchaseId } = context.params;
 
-	const user = userFactory();
-	const isLoggedOut = ! user.get();
+	const isLoggedOut = ! isUserLoggedIn( context.store.getState() );
```

Comment:
Can you move this statement one line down and use the `state` variable?

Reply:
Sure, done in c2bb0c5f7413926db2a9c6121e8a617a37fa3a0b